cmake-edit-core/src/main/java/ru/urururu/cmakeedit/core/checker/Checker.java

```diff
@@ -5,24 +5,22 @@
 
 import java.util.*;
 import java.util.function.Function;
-import java.util.stream.Collectors;
 
 import static com.codahale.metrics.MetricRegistry.name;
 
 /**
  * Created by okutane on 11/08/16.
  */
 public class Checker {
+    static boolean STRICT_MODE = false;
+
     static final Map<String, Function<CommandInvocationNode, Node>> setters =
             new HashMap<String, Function<CommandInvocationNode, Node>>() {{
                 put("set", cmd -> {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove useless curly braces around statement and then remove useless return keyword [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1602)